airflow/providers/google/cloud/operators/automl.py

```diff
@@ -558,7 +558,7 @@ class AutoMLTablesUpdateDatasetOperator(BaseOperator):
     :type gcp_conn_id: str
     """
 
-    template_fields = ("dataset", "update_mask", "location", "project_id")
+    template_fields = ("dataset", "update_mask", "location")
```

Comment:
Unrelated change?

Reply:
I will rebase, it was fixed in other PR.